orttraining/orttraining/models/runner/pipeline.h

```diff
@@ -94,6 +95,14 @@ class PipelineSlot {
     return false;
   }
 
+  bool HasCommute() const {
+    for (auto& task: tasks_) {
+      if (task.IsCommute())
+        return true;
+    }
+    return false;
```

Comment:
nit: might be more concise to use std::any_of  [http://www.cplusplus.com/reference/algorithm/any_of/](http://www.cplusplus.com/reference/algorithm/any_of/)

Reply:
Will change all similar places to std::any_of

---
In reply to: [483783632](https://github.com/microsoft/onnxruntime/pull/5054#discussion_r483783632) [](ancestors = 483783632)